src/View/Helper/CalendarHelper.php

```diff
@@ -183,11 +184,12 @@ public function render() {
 		$month = $this->_View->get('_calendar')['month'];
 
 		$data = $this->dataContainer;
+		$now = new FrozenTime(null, $this->getConfig('timezone', date_default_timezone_get()));
```

Comment:
Why cannot we Pass null here? The Code will use this automatically then internally.